addons/web/static/src/js/views/kanban/kanban_view.js

```diff
@@ -112,6 +113,20 @@ var KanbanView = BasicView.extend({
             this.loadParams.groupBy = [this.arch.attrs.default_group_by];
         }
     },
+    /**
+     * @override
+     * @private
+     * @param {Object} node
+     * @param {Object} fv the fieldsView
+     */
+    _processNode(node, fv) {
+        if (node.tag === 'img') {
+            var dependency_dict = BasicFields.FieldBinaryImage.prototype.fieldDependencies;
+            var fieldsInfo = fv.fieldsInfo[fv.type];
+            fieldsInfo['__last_update'] = _.extend({}, dependency_dict);
+        }
+        return this._super(...arguments);
+    }
```

Comment:
This isn't the correct place to do that: here, the data has already been loaded once. I suggest to detect the presence of such a node (actually, a node using the kanban_image function to determine the src attribute), in kanban_view.js. The arch is processed there, and this is where we determine the fields to load (see processArch) in basic_view.js.

Reply:
@aab-odoo  I have added field dependencies from the _processNode function of basic view when tag img and t-att-src.
https://github.com/odoo/odoo/pull/58313/files#diff-99ee897c28d1aed617dd4302870145bc53bacfd4d9a763a3952b5ab81eba6377R385
cc @mba-odoo

Reply:
I think that this is only necessary for the kanban view, so I'd put this in kanban_view.js

Reply:
@aab-odoo _processNode is called for each node in processArch. _processNode adds field dependency
cc@mba

Reply:
@aab-odoo Changes done :+1: